tests/test_incidents.py

```diff
@@ -227,7 +227,7 @@ def test_add_remove_alert_to_incidents(db_session, setup_stress_alerts_no_elasti
 def test_get_last_incidents(db_session, create_alert):
```

Comment:
can we add a test for deleting an incident and making sure that it gets the right value?

Reply:
We have `test_incident_bl_delete_incident` where we checking the status (reversed, but still )